src/packages/@support/features/feedback-section/layout/FeedbackPost.svelte

```diff
@@ -252,18 +236,19 @@
         />
 
         <div class="d-flex align-items-center gap-2">
-          <!-- <AttachmentIcon
+          <AttachmentIcon
             height={"12px"}
             width={"12px"}
             color={"var(--text-secondary-200)"}
-          /> -->
+          />
```

Comment:
comment this, will uncomment when click behaviour will added

Reply:
Resolved